### PR TITLE
feat(commands): open dev tools

### PR DESCRIPTION
**Pull request by huylg**
Hi, I think opening dev tools directly is more convenient than copying profile url and opening browser manually.

### CODE REVIEW

**Comment by huylg** [comment]
hi @akinsho, please take a look at my pr :D

File: lua/flutter-tools/commands.lua [hunk 1]
```diff
@@ -223,6 +223,11 @@ function M.copy_profiler_url()
   end
 end
 
+function M.open_dev_tools(quiet)
+  send("open_dev_tools",quiet)
```
Comment (akinsho) [inline lua/flutter-tools/commands.lua:227]: @huylg this isn't handled in the debug runner, so anyone running this project via dap/the debugger won't be able to open this. Can you have a look in that file to see if there is command to trigger the devtools via dap

File: lua/flutter-tools.lua [hunk 2]
```diff
@@ -26,6 +26,12 @@ local function setup_commands()
     "FlutterCopyProfilerUrl",
     [[lua require('flutter-tools.commands').copy_profiler_url()]]
   )
+  utils.command(
+    "FlutterOpenDevTools",
+    [[lua require('flutter-tools.commands').open_dev_tools()]]
+  )
+
+  --pub
```
Comment (akinsho) [inline lua/flutter-tools.lua:34]:
```suggestion
  --- Pub
```

**Review by akinsho — COMMENTED** [review]
Generally LGTM I've left a comment about trying to maintain feature parity between the job and debugger runner, so they both can open the dev tools. It's not a hard blocker if you can't find a command for it but I think testing some stuff out a bit would be good

File: lua/flutter-tools/commands.lua (hunk 1 above)
Comment (huylg) [reply lua/flutter-tools/commands.lua:227]: I got it @akinsho, thanks for reviews.

**Pull request merged**